frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -948,6 +867,128 @@ function ReaderBookmark:renameBookmark(item, from_highlight)
     self.input:onShowKeyboard()
 end
 
+function ReaderBookmark:onSearchBookmark(bm_menu)
+    local input_dialog
+    local check_button_case, separator, check_button_bookmark, check_button_highlight, check_button_note
+    input_dialog = InputDialog:new{
+        title = _("Search bookmarks"),
+        input_hint = _("(containing text)"),
```

Comment:
@Frenzie : fine with this input_hint ? (I guess it reads as a continuation `Search bookmarks ... containing text` - or does this work as a name for what's to be inputed ? (containing text vs contained text ?)

Reply:
I'm not sure I know what "containing text" is even supposed to mean, so no. ;-)

It could be left empty, or maybe something like "search phrase".

Reply:
Yes, I would just leave it empty just like in all the other blatantly obvious search dialogs. ^_^
```suggestion
```